src/api/useRemediationsQuery.js

```diff
@@ -97,6 +98,7 @@ const useRemediationsQuery = (
           error: queryError,
           loading: queryLoading,
           meta: metaData,
+          availableIDs: availableIDs,
```

Comment:
Line 85 is what you are looking for. ;) The `fetchAllIds` function can be passed directly to the Table options as `itemIdsInTable`.

Reply:
oh, now I get what you want.